charm/charmcraft.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+name: repo-policy-compliance
+
+type: charm
+
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+summary: Check the repository setup for policy compliance
+
+description: |
+  Used to check whether a GitHub repository complies with expected policies.
```

Comment:
Would attaching this link
https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#repository-setup
help? :D

Reply:
I am not sure if this is applicable here - there are so many things listed there that are not enforced by the policy check (e.g. squash commit, 2 approvals, etc). I do agree that docs should be added though - but I think we really need to review the charm and also the required github token permissions first, and so would suggest this is out of scope for this PR. Your thoughts, @gregory-schiano  ?

Reply:
Yeap I agree, this links corresponds to our own conventions which might differ from what this project enforce